### ISSUE TITLE
`ruby bg2md.rb -b -c -e -r -v NIV Jn 3.16-17 > passage.md` in the README.md does not work

### ISSUE THREAD

**Issue by inteli5** [jgclark/BibleGateway-to-Markdown#25]
Should it be `ruby bg2md.rb -b -c -e -r -v NIV 'Jn 3.16-17' > passage.md`?
If we do `ruby bg2md.rb -b -c -e -r -v NIV Jn 3.16-17 > passage.md`, ref = ARGV[0] will be `Jn` instead of 1`Jn 3.16-17`, right?

**Comment by jgclark** [comment]
Thanks for spotting this documentation bug. I've updated the README accordingly.
